cloudbuild/trigger-tests.yaml

```diff
@@ -19,8 +19,8 @@ steps:
       - |
         curl -sSL https://install.python-poetry.org | python3 - && \
         export PATH="/builder/home/.local/bin:$$PATH" && \
-        make setup && \
-        make test-trigger
+        make install && \
```

Comment:
@roberta-dt rebase your branch on top of `develop` to get the latest changes - you then won't need to update this CI pipeline